pkg/awsManager/ebsManager.go

```diff
@@ -40,40 +41,39 @@ func ListEbsSnapshotForDeletion(client clientpkg.Client) []*string {
 	return ebsSnapshotsToBeDeleted
 }
 
-// this deletes the Ebs Snapshot
+// DeleteEbsSnapshots deletes the Ebs Snapshot
 // successful execution returns nil. Unsuccessful execution or errors occured, would return an error
-func DeleteEbsSnapshots(client clientpkg.Client, ebsSnapshotsToBeDeleted []*string) error {
+func DeleteEbsSnapshots(client clientpkg.Client, ebsSnapshotsToBeDeleted []*string, logger logr.Logger) error {
 
 	if ebsSnapshotsToBeDeleted == nil {
 		return nil
 	}
 	var ebsSnapshotsNotDeleted []*string
-	for _, ebsSnapshotId := range ebsSnapshotsToBeDeleted {
+	for _, ebsSnapshotID := range ebsSnapshotsToBeDeleted {
 
-		_, ebsSnapshotDeleteError := client.DeleteSnapshot(&ec2.DeleteSnapshotInput{SnapshotId: ebsSnapshotId})
+		_, ebsSnapshotDeleteError := client.DeleteSnapshot(&ec2.DeleteSnapshotInput{SnapshotId: ebsSnapshotID})
 		if ebsSnapshotDeleteError != nil {
-			fmt.Println(ebsSnapshotDeleteError)
-			fmt.Print("Could not delete snapshot :", *ebsSnapshotId)
-			ebsSnapshotsNotDeleted = append(ebsSnapshotsNotDeleted, ebsSnapshotId)
+			logger.Error(ebsSnapshotDeleteError, "Failed to delete snapshot", *ebsSnapshotID)
+			ebsSnapshotsNotDeleted = append(ebsSnapshotsNotDeleted, ebsSnapshotID)
 		}
 	}
 
 	if ebsSnapshotsNotDeleted != nil {
-		return errors.New("Could not delete all snapshots for this region")
+		return errors.New("FailedComprehensiveSnapshotDeletion")
 	}
 
 	return nil
 }
 
-func ListVolumeForDeletion(client clientpkg.Client) []*string {
+func listVolumeForDeletion(client clientpkg.Client, logger logr.Logger) []*string {
 
 	var token *string
 	var ebsVolumesToBeDeleted []*string
 
 	for {
 		ebsVolumeList, err := client.DescribeVolumes(&ec2.DescribeVolumesInput{NextToken: token})
 		if err != nil {
-			fmt.Print(err)
+			logger.Error(err, "Failed to retrieve Volume list")
```

Comment:
Maybe, "Failed to list volumes" ?

Reply:
I think "retrieve volume list" is more true to what is happening but am willing to change it if you think it better.

Reply:
Just trying to keep all the errors consistent